File: Hello/app/src/main/java/com/ddam40/example/hello/MainActivity.java
package com.ddam40.example.hello;

import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

  private int clickCount = 0;

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_main);

    Toast.makeText(getApplicationContext(), "프로그래밍 시작", Toast.LENGTH_LONG).show();

    findViewById(R.id.button).setOnClickListener(new View.OnClickListener() {

      @Override
      public void onClick(View v) {
        clickCount = clickCount +1;
        Toast.makeText(getApplicationContext(), "clickCount : " + clickCount, Toast.LENGTH_SHORT).show();
      }

    });
  }
}

File: HelloKotlin/app/src/main/java/com/ddam40/example/hellokotlin/MainActivity.kt
package com.ddam40.example.hellokotlin

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnJava.setOnClickListener {
            startActivity(Intent(this@MainActivity, BmiJavaActivity::class.java))
        }

        btnKotlin.setOnClickListener {
            startActivity(Intent(this@MainActivity, BmiKotlinActivity::class.java))
        }
    }
}
